Dockerfile.cross

```diff
@@ -0,0 +1,21 @@
+# This image is meant to enable cross-architecture builds.
+# It assumes the foundry binaries have already been compiled for `$TARGETPLATFORM` and are
+# locatable in `./dist/bin/$TARGETARCH`
+FROM ubuntu:22.04
+
+# Filled by docker buildx
+ARG TARGETARCH
+
+COPY ./dist/bin/$TARGETARCH/* /usr/local/bin/
+
+ENTRYPOINT ["/bin/sh", "-c"]
+
+LABEL org.label-schema.build-date=$BUILD_DATE \
```

Comment:
do we still need the Dockerfile https://github.com/jsvisa/foundry/blob/docker-arm64/Dockerfile with this approach?

Reply:
Just like reth, I think Dockerfile is also useful, such as someone needs to build the binary in a native way, instead of the `cross build`.

Reply:
yep, makes sense :+1: